src/utils/postUtils.ts

```diff
@@ -0,0 +1,7 @@
+export const convertTimeStringToNumber = (time: string) => {
+  if (time) {
+    const splitedTime = time.split(':');
+    const convertedTime = Number(splitedTime[0]);
+    return convertedTime;
```

Comment:
```suggestion
    return Number(splitedTime[0]);
```

여기도 위처럼 줄여줄수 있을거같습니다!